### PR TITLE
Moved UI documentation to docs folder

### PR DESCRIPTION
**Pull request by KrauseFx**
Ready to be reviewed

### CODE REVIEW

File: fastlane/docs/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 - [Advanced.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Advanced.md#passing-parameters) to show how to pass parameters to lanes from the command line.
 - [Android.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Android.md) Getting started with fastlane for Android
 - [Gitignore.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Gitignore.md) Recommended content for your `.gitignore` file
+- [UI.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/UI.md) More information about how to print out text and ask the user for inputs
```
Comment (asfalcone) [inline fastlane/docs/README.md]: I feel like this should be linked to from the contributing.md, not the readme. Documenting the UI class is for people developing fastlane, not using it.

Comment (KrauseFx) [inline fastlane/docs/README.md]: Not sure, why do we explain how to print out things in the contribution guidelines? I don't think it's appropriate

Comment (KrauseFx) [inline fastlane/docs/README.md]: What I'm thinking is that the Contribution guidelines should be about the guidelines we have for people to contribute, not so much about how to implement certain things. Does that make sense?

Comment (asfalcone) [inline fastlane/docs/README.md]: Agreed. Sorry I thought this was going into the fastlane/README not fastlane/docs/README! :+1:

**Comment by asfalcone** [comment]
Good to go :rocket:

**Pull request merged**